clik/external/hal_cpu/source/hal_main.cpp

```diff
@@ -62,16 +62,23 @@ class cpu_hal_platform : public hal::hal_t {
     const uint64_t local_ram_size = 8 << 20;
     hal_device_info.type = hal::hal_device_type_riscv;
     hal_device_info.word_size = sizeof(uintptr_t) * 8;
-    hal_device_info.target_name = "CPU";
+    hal_device_info.target_name = "ock cpu";
```

Comment:
Is this change related to the PR?

Reply:
it's not absolutely necessary no, but I found it a bit clearer.